gekitchensdk/erd/values/laundry/erd_drynessnew_level.py

```diff
@@ -0,0 +1,10 @@
+import enum
```

Comment:
I think you can remove this file based on the other changes made.

Reply:
done